config/sql/migration/migrate.15.sql

```diff
@@ -0,0 +1,7 @@
+-- Don't modify this! Create a new migration instead--see docs/schema-migrations.adoc
+BEGIN TRANSACTION;
+
+CREATE TABLE migrations(migration TEXT NOT NULL, version INT NOT NULL);
```

Comment:
`migrations` is not enough self-explanatory IMO, as it's only storing backward migrations which are a special case. `version` could also be `version_from` or `version_to` depending on which is it.

We have to be careful about the naming here because we probably won't be able to rename this table later.

Reply:
Also, the version field should probably be the primary key